src/main/resources/view/ReminderCard.fxml

```diff
@@ -6,7 +6,8 @@
 <VBox xmlns="http://javafx.com/javafx/11" xmlns:fx="http://javafx.com/fxml/1" styleClass="reminder-card  ">
    <children>
       <Label styleClass="reminder-description" fx:id="description" text="Text" wrapText="true" />
-      <Label styleClass="reminder-with" fx:id="with" text="with Lorem Ipsum" wrapText="true" />
+      <Label styleClass="reminder-name" fx:id="name" text="Text" wrapText="true" />
+      <Label styleClass="reminder-phone" fx:id="phone" text="Text" wrapText="true" />
```

Comment:
Not sure whether there's a need to display phone number here even though you're using the phone number as a key.